### PR TITLE
Added aws_endpoint_url cli option

### PR DESCRIPTION
**Pull request by fewstera**
To be able to use awslogs with locally running instances of cloudwatch such as [localstack](https://github.com/localstack/localstack), we need a way to overwrite the endpoint url.

This PR adds an `--aws-endpoint-url` cli flag to change the endpoint that is called by awslogs.

### CODE REVIEW

**Comment by fewstera** [comment]
@jorgebastida any chance you could review this please?

**Comment by jonathan-palumbo** [comment]
Would be really great to merge this addition!

**Comment by Glutexo** [comment]
+1

File: awslogs/bin.py [hunk 1]
```diff
@@ -55,6 +55,12 @@ def add_common_arguments(parser):
                             default=os.environ.get('AWS_REGION', None),
                             help="aws region")
 
+        parser.add_argument("--aws-endpoint-url",
```
Comment (jorgebastida) [inline awslogs/bin.py:58]: Making this argument read the environment like the other arguments would be ideal

**Comment by jorgebastida** [comment]
It is such a long time that it is embarrassing, but it somebody manages to do that quick fix I would happily merge it.
Please, remember to add this arg to the the doc too.

**Comment by leandrodamascena** [comment]
> It is such a long time that it is embarrassing, but it somebody manages to do that quick fix I would happily merge it.
> Please, remember to add this arg to the the doc too.

@jorgebastida added PR. Please check if it's enough.

Thank you.

**Pull request merged**